app.rb

```diff
@@ -305,5 +337,9 @@ def maybe_redact_ssn(ssn)
     ssn&.gsub(/\d/, '#')
   end
 
+  def vtr_enabled?
+    ENV['vtr_enabled'] == 'true'
```

Comment:
I think I would expect this to be `VTR_ENABLED` (all caps) to match the oidc pr. But other than that LGTM

Reply:
It looks like the convention is for lower case `ENV` vars in this repo (and in the OIDC one; the `VTR_ENABLED` config is an oddball there).